### PR TITLE
Documentation and description for FluidHeatFlow library improved

### PR DESCRIPTION
**Pull request by TManikantan**
Solves #3474

### CODE REVIEW

**Comment by TManikantan** [comment]
@christiankral kindly review the updates done.

**Comment by beutlich** [comment]
@TManikantan Please set labels `example` and `L: Thermal.FluidHeatFlow` for a proper categorization! You might check your [onboarding wiki page](https://github.com/modelica/ModelicaStandardLibrary/wiki/Onboarding-MAP-Lib-Project-Leader-and-Deputy) for a motivation.

File: Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo [hunk 1]
```diff
@@ -7,17 +7,17 @@ model ParallelCooling "Cooling circuit with parallel branches"
...
+    prescribedHeatFlow1.port.T-TAmb "Temperature difference between heat source 1 and ambient condition";
   output SI.TemperatureDifference dTtoPipe1=prescribedHeatFlow1.port.T-pipe1.T_q
-    "Source1 over Coolant1";
+    "Temperature difference between heat source 1 and coolant in pipe 1";                      
   output SI.TemperatureDifference dTCoolant1=pipe1.dT
-    "Coolant1's temperature increase";
+    "Coolant1's temperature increase in pipe 1";
   output SI.TemperatureDifference dTSource2=
-    prescribedHeatFlow2.port.T-TAmb "Source2 over Ambient";
+    prescribedHeatFlow2.port.T-TAmb "Temperature difference between Heat source 2 and ambient condition";
```
Comment (christiankral) [inline Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo:16]:
```suggestion
    prescribedHeatFlow2.port.T-TAmb "Temperature difference between heat source 2 and ambient condition";
```

File: Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo [hunk 2]
```diff
@@ -169,43 +169,43 @@ Two prescribed heat sources dissipate their heat through thermal conductors to c
...
 <td>dTCoolant2</td>
-<td>Coolant's temperature increase</td>
+<td>Coolant2's temperature increase in pipe 2</td>
 <td>Losses * cp * totalMassFlow/2</td>
 <td>20 K</td>
 </tr>
 <tr>
 <td>dTmixedCoolant</td>
-<td>mixed Coolant's temperature increase</td>
+<td>Mixed Coolant's temperature increase</td>
```
Comment (christiankral) [inline Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo:208]:
```suggestion
<td>Mixed coolant's temperature increase</td>
```

File: Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo (hunk 1 above)
Comment (christiankral) [reply Modelica/Thermal/FluidHeatFlow/Examples/ParallelCooling.mo:16]: Lower case spell of heat

**Review by christiankral — APPROVED** [review]
The changes are good with me now

**Comment by TManikantan** [comment]
> @TManikantan Please set labels `example` and `L: Thermal.FluidHeatFlow` for a proper categorization! Furthermore, a milestone shall be set, too. You might check your [onboarding wiki page](https://github.com/modelica/ModelicaStandardLibrary/wiki/Onboarding-MAP-Lib-Project-Leader-and-Deputy) for a motivation.

Thankyou for your suggestions @beutlich . I will take care of them in the future too.

File: Modelica/Thermal/FluidHeatFlow/Examples/IndirectCooling.mo [hunk 3]
```diff
@@ -173,31 +173,31 @@ Inner coolant's temperature rise near the source is the same as temperature drop
...
 </tr>
 <tr>
 <td>dTtoPipe</td>
-<td>Source over inner Coolant</td>
+<td>Temperature difference between heat source and coolant in pipe 1</td>
 <td>Losses / ThermalConductor.G</td>
 <td>10 K</td>
 </tr>
 <tr>
 <td>dTinnerColant</td>
```
Comment (arunkumar-narasimhan) [inline Modelica/Thermal/FluidHeatFlow/Examples/IndirectCooling.mo:187]: Side note, dTinnerCoolant has been misspelled. Would be good to correct it.

File: Modelica/Thermal/FluidHeatFlow/Examples/IndirectCooling.mo [hunk 4]
```diff
@@ -173,31 +173,31 @@ Inner coolant's temperature rise near the source is the same as temperature drop
...
 </tr>
 <tr>
 <td>dTCooler</td>
-<td>Cooler's temperature rise between inner and outer pipes</td>
+<td>Coolant temperature difference between inner pipe and outer pipe</td>
 <td>Losses * (innerGc + outerGc)</td>
 <td>10 K</td>
 </tr>
 <tr>
 <td>dTouterColant</td>
```
Comment (arunkumar-narasimhan) [inline Modelica/Thermal/FluidHeatFlow/Examples/IndirectCooling.mo:199]: Side note, dTouterCoolant has been misspelled. Would be good to correct it.

**Review by arunkumar-narasimhan — APPROVED** [review]
Looks good to me now.

**Review by MartinOtter — APPROVED** [review]

**Pull request merged**